### PR TITLE
Minor docs tweaks

### PR DESCRIPTION
**Pull request by ewels**
Small stuff I noticed whilst making the new cookiecutter thing.

### CODE REVIEW

**Comment by ewels** [comment]
Ping @Hammarn

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -34,28 +34,6 @@ Samples that receive less than 5% alignment are skipped for further analysis. Yo
 ### Data organization
 The pipeline can't take a list of multiple input files - it takes a glob expression. If your fastq files are scattered in different paths then we recommend that you generate a directory with symlinked files. If running in paired end mode please make sure that your files are sensibly named so that they can be properly paired. See the previous point.
 
-## UPPMAX specific options
```
Comment (Hammarn) [inline docs/troubleshooting.md]: Why did you remove this section?

Comment (ewels) [reply docs/troubleshooting.md]: Because we don't need to do `module load bioinfo-tools` any more, since we are using singularity.

Comment (Hammarn) [reply docs/troubleshooting.md]: Well why do we even have an uppmax config file then? Seem very arbitrary to just remove some documentation but not the actual feature.

**Comment by Hammarn** [comment]
👏
